docs/index.asciidoc

```diff
@@ -7,3 +7,4 @@
 include::{asciidoc-dir}/../../shared/versions70.asciidoc[]
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
 include::k8s-quickstart.asciidoc[]
+include::accessing-services.asciidoc[]
```

Comment:
Will this not include the new doc on the same page as the quickstart, given that we build with `--single`? Let me know if I got my asciidoc all wrong but is it not time to have two separate documents now and an index page linking to them?

Reply:
We are going to have two separate docs and an index. Will share a preview asap.